assets/js/translator.js

```diff
@@ -13,6 +13,8 @@ var UPLOAD_FILE_SIZE_LIMIT = 32E6,
     TRANSLATION_LIST_ROWS = 8,
     TRANSLATION_LIST_COLUMNS = 4;
 
+var INDIVIDUAL_LIST_WIDTH = TRANSLATION_LIST_WIDTH / TRANSLATION_LIST_COLUMNS;
```

Comment:
Computed values should be normal case. Uppercase is reserved for constants that are set by the user.